src/Location.ml

```diff
@@ -0,0 +1,62 @@
+open Base
+open Lexing
+
+type t =
+  | NoSource
+  | Source of {
+      filename : string; [@equal.ignore]
+      start_line : int; [@equal.ignore]
+      start_column : int; [@equal.ignore]
+      end_line : int; [@equal.ignore]
+      end_column : int; [@equal.ignore]
+    }
+[@@deriving equal, sexp]
```

Comment:
Since you are ignoring everything except the constructors, do we even need equality for this type?

Reply:
Done